admin-api/authorization_server/external_authorization.md

```diff
@@ -0,0 +1,161 @@
+---
+title: External authorization server
+menuTitle: External authorization server
+weight: 2
+---
+
+# External authorization server
+
+You can use external authorization server to access your resources, to do that you need to install a module that implements our `PrestaShop\PrestaShop\Core\Security\OAuth2\AuthorisationServerInterface` interface.
+
+As an example we developed a [keycloak_connector_demo module](https://github.com/PrestaShop/keycloak_connector_demo) based on the [Keycloak authorization server](https://www.keycloak.org/).
+
+This module includes docker configuration that allows you to launch a keycloak server locally for development purposes. It includes:
+- Keycloak authorization server and its admin interface
+- One default client
+- Some default scopes matching the ones from the core
+
+{{% notice note %}}
+Any authorization server can interact and authorize the Admin API thanks to this implementation. However, both need a common contract which is based on the **scopes** defined on each endpoint of the resource server. So you will need to provide your authorization server with a list of scopes mapping the ones used by the endpoints.
+{{% /notice %}}
+
+## Keycloak server initialization
+
+To start the docker container run this command from the root folder of the module:
+
+```bash
+docker compose up
+# OR if you want keycloak to keep running in background
+docker compose up -d
+```
+
+You will then have access to the server administration via `http://localhost:8003` where you will find a realm named `prestashop`
+User: `admin`
+Password: `admin`
+
+## Module installation and configuration
+
+Clone the module repository into your `modules` folder, then go to the Module manager in your back office and install the `Keycloak OAuth2 connector demo` module.
+
+Then go to the configuration and provide the two needed urls for configuration:
+
+- **Keycloak realm endpoint**: the url on which your server can access the keycloak realm endpoint, this is the base realm URL from which the connector can fetch the certificates and validate if the JWT Token is valid
+- **Keycloak allowed issuers**: A list of issuer values that are considered valid, usually it's the same as the realm endpoint But in case you request your access token from different urls you can specify multiple endpoints so the validation process accepts multiple issuers.
```

Comment:
```suggestion
- **Keycloak allowed issuers**: A list of issuer values that are considered valid, usually it's the same as the realm endpoint, but in case you request your access token from different URLs, you can specify multiple endpoints, so the validation process accepts multiple issuers.
```